packages/api/cms-api/src/generator/generate-crud.ts

```diff
@@ -83,6 +83,19 @@ function buildOptions(metadata: EntityMetadata<any>) {
         return hasFieldFeature(metadata.class, prop.name, "input") && prop.type === "RootBlockType";
     });
 
+    const mainProps = metadata.props.filter((prop) => {
```

Comment:
This probably should be renamed to rootProps

Reply:
ops